Changelog.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0
+
+*Support Unicode version 9.0
```

Comment:
Need a space after `*`. Also, can you update `package.yaml` as well with version "9.0"? Once you do that you will have to do a stack build which will generate the .cabal file. Then commit both package.yaml and .cabal.